Se 8193 cache scope

Cristian-Venticinque opened the pull request.

Commit:
Added note clarifying how to add the component in the XML file.

Commit:
Added Example of Invalidate Cache usage.

Commit:
Changed Title name for disambiguation

elrodro83 commented on modules/ROOT/pages/cache-scope.adoc:
```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
+----
+<flow name="invalidateCache">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-cache cachingStrategy-ref="InvalidatableCachingStrategy" />
```
you may want to also document the `keyGenerator-ref ` and `keyGenerationExpression ` attributes. Example: https://github.com/mulesoft/mule-ee/blob/mule-3.x/tests/cache/src/test/resources/config/invalidate-cache-key-config.xml

Cristian-Venticinque replied:
Changes addressed in commit e8e7f09. Added more XML samples.

elrodro83 commented on modules/ROOT/pages/cache-scope.adoc:
```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
+
+To invalidate a cache, you can create a Flow with two components:
+
+* An `http:listener` component that will be the entry point of the Flow.
```
`An` -> `A`

dhirsch1 replied:
An is correct because of the way the h is pronounced.

dhirsch1 commented on modules/ROOT/pages/cache-scope.adoc:
```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
+
+=== Example: Invalidating Cache Usage
+
+To invalidate a cache, you can create a Flow with two components:
```
Remove "you can"

dhirsch1 commented on modules/ROOT/pages/cache-scope.adoc:
```diff
@@ -385,7 +385,31 @@ There are two message processors for invalidating caches:
+* An `http:listener` component that will be the entry point of the Flow.
+* An `ee:invalidate-cache` component that will invalidate the referenced cache.
+
+Once this Flow gets executed, it invalidates the referenced cache clearing all its stored entries.
```
Change "gets" to "is."

"...by clearing all of its stored entries."

dhirsch1 replied:
"is executed" instead of "gets executed."

"...by clearing all of its stored entries."

Commit:
Addressed changes requested during review.

-Improved wording to be aligned with our style. 
-Added XML samples illustrating how to invalidate a cache key.

elrodro83 commented on modules/ROOT/pages/cache-scope.adoc:
```diff
@@ -385,7 +385,61 @@ There are two message processors for invalidating caches:
+----
+<flow name="invalidateKey">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-key cachingStrategy-ref="InvalidatableCachingStrategy" keyGenerator-ref="keyGenerator"/>
```
the definition of the keyGenerator is needed to understand this example

Commit:
Added missing `keyGenerator` bean definition.

Commit:
Removed incorrect usage of *bold*

Commit:
Added `backticks` to values and object names

Commit:
Removed links within the doc. Moved inline xrefs to See Also.

Commit:
Removed redundant sentence pointing to a link whithin the article

Commit:
Removed external link

Commit:
Changed "Flow" to lowercase.

kevintroller reviewed (commented):
LGTM!
